### ISSUE TITLE
error: NU1108: Cycle detected.

### ISSUE THREAD

**Issue by celsojr** [SciSharp/LLamaSharp#27]
I have tried all versions in both dotnet 6 and 7 and got the same error.
I'm on Windows 11.
Am I missing something else??
![image](https://github.com/SciSharp/LLamaSharp/assets/6450642/42be1987-bf52-4a85-8f2d-27d5d10e89ea)

**Comment by AsakusaRinne** [comment]
Hi, the image you uploaded was broken. Could you please provide some more information?

**Comment by celsojr** [comment]
This is a fresh new dotnet 6 console app, but I'm not able to install this package :(
```
PS C:\Users\[...]\LLamaSharp> dotnet add .\LLamaSharp\LLamaSharp.csproj package LLamaSharp --version 0.3.0
  Determining projects to restore...
  Writing C:\Users\[...]\AppData\Local\Temp\tmpdo5h4v.tmp
info : X.509 certificate chain validation will use the default trust store selected by .NET for code signing.
info : X.509 certificate chain validation will use the default trust store selected by .NET for timestamping.
info : Adding PackageReference for package 'LLamaSharp' into project '.\LLamaSharp\LLamaSharp.csproj'.
info : Restoring packages for C:\Users\[...]\LLamaSharp\LLamaSharp.csproj...
error: NU1108: Cycle detected.
error:   LLamaSharp -> LLamaSharp (>= 0.3.0).
info : Package 'LLamaSharp' is compatible with all the specified frameworks in project '.\LLamaSharp\LLamaSharp.csproj'.
error: Value cannot be null. (Parameter 'path1')


Usage: NuGet.CommandLine.XPlat.dll package add [options]

Options:
  -h|--help               Show help information
  --force-english-output  Forces the application to run using an invariant, English-based culture.
  --package               Id of the package to be added.
  --version               Version of the package to be added.
  -d|--dg-file            Path to the dependency graph file to be used to restore preview and compatibility check.
  -p|--project            Path to the project file.
  -f|--framework          Frameworks for which the package reference should be added.
  -n|--no-restore         Do not perform restore preview and compatibility check. The added package reference will be unconditional.
  -s|--source             Specifies NuGet package sources to use during the restore.
  --package-directory     Directory to restore packages in.
  --interactive           Allow the command to block and require manual action for operations like authentication.
  --prerelease            Allows prerelease packages to be installed.
```

**Comment by AsakusaRinne** [comment]
Hi, it seems that you created a project using the same name "LLamaSharp" ([reference](https://github.com/dotnet/BenchmarkDotNet/issues/2005)). Could you please have a project with different name and then have an another try?

**Comment by celsojr** [comment]
Now this is working fine, and it's pretty fast! 🥳🎉

What I did was, I created a Python project just to try the model out and make sure it would work fine with similar setup on my machine.

Then, with c#, I create a new `Class Library` with the `NetStandard 2.0` and installed the both `LLamaSharp 0.4.0` and `LLamaSharp.Backend.Cpu 0.3.1` in this project.

Then I created a new separated project with the `net7.0` and put a reference for the `NetStandard` project calling a method over there.

Then I realized that the problem was just the name of the project as you said. 🤡

THAAAAAAAAAAAAANKS! 😁

![image](https://github.com/SciSharp/LLamaSharp/assets/6450642/0ab61a41-ef0d-4438-8af8-aa8f1972631f)
